JSDemos/Demos/Charts/CustomizePointsAndLabels/description.md

```diff
@@ -1 +1,20 @@
-This demo shows how you can customize individual points and labels in&nbsp;the Chart.
\ No newline at end of file
+This demo shows how you can customize individual points and labels in the Chart component.
+
+## Customize Points
+
+Use the [customizePoint](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#customizePoint) function to change the appearance of individual series points. This function should return an object with properties that you want to change in a [point](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/point/) configuration. This demo uses this function to color all the points above the high average red, and the points below the low average blue.
```

Comment:
`This demo uses this function to color all the points above the high average red, and the points below the low average blue.`

I'd like to add a noun after "high average" that describes this dotted line. For example, is it technically correct to say "high average marker (red)" or "high average line (red)"?
Please clarify this for "low average" as well.

Reply:
I rewrote this sentence a bit, could you please have a look?

Reply:
Thanks, Vlada. I added a suggestion to update your new version. Please take a look.